Formula/rxvt-unicode.rb

```diff
@@ -31,6 +32,11 @@ class RxvtUnicode < Formula
 
   uses_from_macos "perl"
 
+  resource "libptytty" do
+    url "https://github.com/yusiwen/libptytty/archive/b9694ea18e0dbd78213f55233a430325c13ad63e.tar.gz"
```

Comment:
will wait for the tagged release in the upstream.

Reply:
We should consider making this its own formula if it gets a release tag.

Reply:
Agree.